contrib/runners/windows_runner/windows_runner/windows_script_runner.py

```diff
@@ -195,7 +195,7 @@ def _get_script_arguments(self, named_args=None, positional_args=None):
                         cmd_parts.append('-%s' % (arg))
                     else:
                         cmd_parts.append('-%s:$false' % (arg))
-                elif isinstance(value, (list, tuple)) or hasattr(value, '__iter__'):
+                elif isinstance(value, (list, tuple)):
```

Comment:
Probably for no good reason.

This won't work, because in Python 3, `string` type also implements that interface and has `__iter__` method.

One option would be to do like `if not isinstance(value, six.string_types) and hasattr(value, '__iter__')` but that's probably an overkill and I believe there is actually no need to check for `__iter__` here because those values come from user via API and only simple types such as arrays and strings are valid there.